src/vehicle.h

```diff
@@ -714,7 +714,38 @@ class vehicle
         // Vehicle parts descriptions - descriptions for all the parts on a single tile
         void print_vparts_descs( const catacurses::window &win, int max_y, int width, int &p,
                                  int &start_at, int &start_limit ) const;
-
+        // owner functions
+        inline void set_old_owner( const faction *temp_owner ) {
```

Comment:
Functions defined inside a class definition are automatically `inline`. The `inline` keyword here is redundant.
```suggestion
        void set_old_owner( const faction *temp_owner ) {
```
Same for all the functions below.